log_injection/Rahul-Sahani04_MikoChan-Telegram-Bot-V2/V2.py

```
@@ -0,0 +1,128 @@
+    try:
+        response = model.send_message("\n".join(context_messages) + "\n.The Message to reply to:" + message, safety_settings=safety_settings)
+        return response.text
+    except Exception as e:
+        return f"Sorry, I couldn't understand that. Error: {str(e)}"
+
+# Function to get a reply from Gemini API for image messages
+def get_image_gemini_reply(message):
+    try:
+        response = model.send_message([f"{"\n".join(context_messages)}\n- The Message to reply to is the image.", message], safety_settings=safety_settings)
+        response.resolve()
+        return response.text
```

Ruff (0.13.3)

Consider moving this statement to an `else` block

(TRY300)